### PR TITLE
Fix Sidebar Bugs

### LINKED ISSUE #7: Auto resize textbox isn't scrollable after it hits max size
image
I cannot scroll down to view the rest of the entry.

### PR DESCRIPTION
**Pull request by Alex967CW**
- fixed dark mode issue
- fixed time zone bug
- fixed height + highlighting bugs
- fixed #7

### CODE REVIEW

File: src/pages/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -21,49 +21,52 @@ export default function Sidebar({ element }: { element: React.ReactElement; }) {
...
+    <div className="flex w-full">
+      <div
+        id="docs-sidebar"
+        className="scrollbar-y scroll-hidden bottom-0 left-0 top-0 z-[60] h-screen min-w-[320px] max-w-[320px] transform overflow-y-auto border-gray-200 bg-light_yellow transition-all duration-300 lg:bottom-0 lg:right-auto lg:block lg:translate-x-0"
+      >
         <div className="flex h-full flex-col">
           <div className="m-4 flex grow flex-col">
             <div className="m-4 flex grow flex-col">
-              <div className="mb-1 text-sm font-semibold text-gray-600">
+              <div className="mb-1 text-sm font-semibold text-dark_gray">
```
Comment (jonathanjma) [inline src/pages/Sidebar/Sidebar.tsx:37]: dont do this we replaced dark_gray with gray-600 in the tailwind config

File: src/pages/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -21,49 +21,52 @@ export default function Sidebar({ element }: { element: React.ReactElement; }) {
...
-                  <img
-                    className="mx-auto block max-h-[40px] max-w-[40px] justify-center rounded-full sm:mx-0 sm:shrink-0"
-                    src={user!.profile_picture}
-                    alt="profile"
-                  />
-                </div>
-                <div className="mr-2 text-secondary">
-                  <div className="text-base font-semibold">
-                    {user!.username}
+              <a className="flex-none" href={"/profile/" + user!.id}>
```
Comment (jonathanjma) [inline src/pages/Sidebar/Sidebar.tsx:40]: adding the <a> breaks the logout button. also design changed the way this works to a dropdown right?

File: src/pages/Sidebar/Sidebar.tsx [hunk 3]
```diff
@@ -21,49 +21,52 @@ export default function Sidebar({ element }: { element: React.ReactElement; }) {
...
+                      alt="profile"
+                    />
                   </div>
-                  <div className="text-xs text-gray-400">
-                    Logging since {user!.created.substring(0, 4)}
+                  <div className="mr-2 text-secondary">
+                    <div className="text-base font-semibold">
+                      {user!.username}
+                    </div>
+                    <div className="text-xs text-light_gray">
```
Comment (jonathanjma) [inline src/pages/Sidebar/Sidebar.tsx:56]: again use gray-400 instead of light_gray

File: src/pages/Sidebar/Sidebar.tsx [hunk 4]
```diff
@@ -77,14 +80,11 @@ export default function Sidebar({ element }: { element: React.ReactElement; }) {
                           href={entry.route}
                           selectedClass={[
                             "bg-yellow font-semibold text-secondary shadow-md1 ",
-                            "bg-light_yellow font-medium text-gray-600 ",
+                            "bg-light_yellow font-medium text-dark_gray ",
```
Comment (jonathanjma) [inline src/pages/Sidebar/Sidebar.tsx:83]: no dark_gray

**Review by jonathanjma — CHANGES_REQUESTED** [review]

**Review by zachseidner1 — CHANGES_REQUESTED** [review]
LGTM. Just fix the styling issue I mentioned and you should good to go.

File: src/pages/Sidebar/Sidebar.tsx [hunk 5]
```diff
@@ -21,69 +23,85 @@ export default function Sidebar({ element }: { element: React.ReactElement; }) {
...
+                <></>
+              )}
               <HappinessForm height={height} />
               <div className="flex grow flex-col">
                 <nav className="relative w-full grow">
-                  {navConfig.map((entry, index) => {
+                  {navConfig.map((entry) => {
                     return (
                       <div key={entry.route} className={"w-full"}>
                         <LinkButton
```
[the remark this answers is not in the document]
Comment (Alex967CW) [reply src/pages/Sidebar/Sidebar.tsx:95]: should be fixed in the next commit

File: src/pages/Sidebar/Sidebar.tsx [hunk 6]
```diff
@@ -21,69 +23,85 @@ export default function Sidebar({ element }: { element: React.ReactElement; }) {
...
 
   return (
-    <div className="flex h-screen w-full">
-      <div className="scrollbar-y bottom-0 left-0 top-0 z-[60] hidden min-w-[320px] max-w-[320px] transform overflow-y-auto border-gray-200 bg-light_yellow transition-all duration-300 lg:bottom-0 lg:right-auto lg:block lg:translate-x-0">
+    <div className="flex w-full">
+      <div
+        id="docs-sidebar"
+        className="scrollbar-y scroll-hidden bottom-0 left-0 top-0 z-[60] h-screen min-w-[320px] max-w-[320px] transform overflow-y-auto border-gray-200 bg-light_yellow transition-all duration-300 lg:bottom-0 lg:right-auto lg:block lg:translate-x-0"
+      >
         <div className="flex h-full flex-col">
```
[the remark this answers is not in the document]
Comment (Alex967CW) [reply src/pages/Sidebar/Sidebar.tsx:34]: yeah, I understand. I've been using them on the Statistics page.

File: src/pages/Sidebar/Sidebar.tsx (hunk 2 above)
Comment (Alex967CW) [reply src/pages/Sidebar/Sidebar.tsx]: added dropdown so this should be good, and logout seems to work

**Review by zachseidner1 — APPROVED** [review]
Thanks for responding to my comments so quickly! You're good to merge.

**Pull request merged**